charts/nodejs/templates/hpa.yaml

```diff
@@ -8,10 +8,11 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    sandboxId: "{{ .Values.biomageCi.sandboxId }}"
 spec:
   scaleTargetRef:
     kind: Deployment
-    name: {{ template "appname" . }}
+    name: {{ template "appname_with_ref" . }}
```

Comment:
This was a bug, the deployment is named using `appname_with_ref`, so the HPA wouldn't have worked if we turned it on.